packages/spor-alert-react-native/src/ClosableAlert.tsx

```diff
@@ -0,0 +1,43 @@
+import { CloseOutline18Icon } from "@vygruppen/spor-icon-react-native";
+import React, { useEffect, useState } from "react";
+import { Box } from "@vygruppen/spor-layout-react-native";
+import { Text } from "@vygruppen/spor-typography-react-native";
+import { Pressable } from "react-native";
+import { BaseAlert, ColorVariants } from "./BaseAlert";
+
+type ClosableAlertProps = {
+  children: React.ReactNode;
+  color: ColorVariants;
+  icon: JSX.Element;
+  title: string;
```

Comment:
Må en `title` være spesifisert for `ClosableAlert`s? Eller skal man kunne bruke dem med bare `children`?

Reply:
Her er jeg litt usikker, men slik jeg tolker [Figma](https://www.figma.com/file/Tmr2URVX2vNkyRLqKhNRQA/Vy_komponentbibliotek?node-id=3230%3A36130) ser det ut som `ClosableAlert` skal ha en `title` 🤔